modules/lang-painless/src/main/java/org/elasticsearch/painless/node/ENewArrayFunctionRef.java

```diff
@@ -48,17 +49,18 @@ public ENewArrayFunctionRef(Location location, String type) {
     }
 
     @Override
-    Output analyze(ScriptRoot scriptRoot, Scope scope, Input input) {
-        this.input = input;
-        output = new Output();
+    Output analyze(ClassNode classNode, ScriptRoot scriptRoot, Scope scope, Input input) {
+        FunctionRef ref;
```

Comment:
Consider moving this closer to assignment.

Reply:
Done.